backend/src/main/kotlin/app/ehrenamtskarte/backend/application/database/Schema.kt

```diff
@@ -48,6 +48,7 @@ class ApplicationEntity(id: EntityID<Int>) : IntEntity(id) {
     var status: Status
         get() = _status
         set(newValue) {
+            if (_status == newValue) return
```

Comment:
I assume we don't want to update statusResolvedDate if the status is changed from "approved" to "approved" e.g.